src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = costPerDay * days;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Consider using a constant variable for the discount value 50, e.g., `const DISCOUNT_FOR_7_DAYS = 50;`, to improve code readability and maintainability.